<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NCRDDataSourceTypeDialogUI</class>
 <widget class="QDialog" name="NCRDDataSourceTypeDialogUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Datasource type</string>
  </property>
  <layout class="QHBoxLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QGroupBox" name="gbTypes">
     <property name="title">
      <string>Datasource types</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QRadioButton" name="rbDataView">
        <property name="text">
         <string>DataView</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbSql">
        <property name="text">
         <string>SQL query</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbText">
        <property name="text">
         <string>Text</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbXml">
        <property name="text">
         <string>XML</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbStringList">
        <property name="text">
         <string>String list</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbItem">
        <property name="text">
         <string>Item model</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbCustom">
        <property name="text">
         <string>Custom</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout">
     <item>
      <widget class="QPushButton" name="btnCreate">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="3" margin="6"/>
 <tabstops>
  <tabstop>rbSql</tabstop>
  <tabstop>rbText</tabstop>
  <tabstop>rbXml</tabstop>
  <tabstop>rbStringList</tabstop>
  <tabstop>rbItem</tabstop>
  <tabstop>rbCustom</tabstop>
  <tabstop>btnCreate</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnCreate</sender>
   <signal>clicked()</signal>
   <receiver>NCRDDataSourceTypeDialogUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>NCRDDataSourceTypeDialogUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
